packages/eslint-plugin-pf-codemods/src/rules/v6/formGroupRenameLabelIcon/formGroup-rename-labelIcon.md

```diff
@@ -0,0 +1,18 @@
+### formGroup-rename-labelIcon [(#10016)](https://github.com/patternfly/patternfly-react/pull/10016)
+
+The `labelIcon` prop for FormGroup has been renamed to `labelHelp`. We recommend using FormGroupLabelHelp element for the labelHelp prop. The markup has also changed, we now wrap the labelHelp element in `<span className={styles.formGroupLabelHelp}>`, so there is no need to add `className={styles.formGroupLabelHelp}` to the labelHelp element.
```

Comment:
I may have not added a comment when I left a review last week, but pretty sure it was for the message relayed to the consumer. Instead of styles.formGroupLabelHelp, feel like we should use the actual class string instead. Would be for here and the tests/actual rule code.